.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -64,11 +64,10 @@ jobs:
       - name: Checkout Source code
         uses: actions/checkout@v2
 
-      - name: Install Rust Toolchain
-        uses: actions-rs/toolchain@v1
-
-      - name: Install WASM target
-        run: rustup target add wasm32-unknown-unknown
+      - name: Install and restore nix
+        uses: ./.github/actions/cache_install_nix
+        with:
+          cache-key: ${{ runner.os }}-nix_store-v4-${{ hashFiles('**/Cargo.lock') }} # bump cache version if you need to clear it, see https://github.com/actions/cache/issues/485 and https://stackoverflow.com/questions/63521430/clear-cache-in-github-actions
```

Comment:
not that it matters much, but is there a reason to start with the cache key versioned as 4 already?

Reply:
This integration is removed from this PR.